build/installEdgeDriver.ps1

```diff
@@ -0,0 +1,19 @@
+$url = "https://msedgedriver.azureedge.net/119.0.2151.72/edgedriver_win64.zip" #Chrome Driver from https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/
+$fileName = "edgedriver_win64.zip"
+$source = "C:\Downloads\$fileName"
+$destination = "C:\Program Files\dotnet\"
+
+if (Test-Path "$PSScriptRoot\win-installer-helper.psm1") 
```

Comment:
would be good to have a try catch and also add some `Write-Verbose` for better logs